README.md

```diff
@@ -667,7 +697,9 @@ Deci provides implementation for various datasets. If you need to download any o
 - [Coco](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/datasets/detection_datasets/coco_detection.py)
 - [PascalVOC 2007 & 2012](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/datasets/detection_datasets/pascal_voc_detection.py)
   
-  
+### Pose Estimation
+
+- [COCO](https://github.com/Deci-AI/super-gradients/blob/cadcfdd64e7808d21cccddbfaeb26acb8267699b/src/super_gradients/recipes/dataset_params/coco_pose_estimation_dekr_dataset_params.yaml)
```

Comment:
The other links map to `.py` file, and here to `.yaml` file, is that on purpose?

Reply:
Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2